src/arch/xtensa/configs/apollolake_defconfig

```diff
@@ -8,3 +8,7 @@ CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_COMP_SRC_SMALL=y
 CONFIG_COMP_TDFB=n
 CONFIG_COMP_TONE=n
+CONFIG_COMP_KPB=n
```

Comment:
btw, we have a weird sof-apl-keyword-detect.m4 which presumably uses KPB on APL, although FWIW my brief test of that topology on UP2 wasn't very successful, so maybe it's broken anyway